Content.Server/MachineLinking/System/SignalButtonSystem.cs

```diff
@@ -12,13 +12,20 @@ public sealed class SignalButtonSystem : EntitySystem
         public override void Initialize()
         {
             base.Initialize();
-
+            SubscribeLocalEvent<SignalButtonComponent, ComponentInit>(OnInit);
             SubscribeLocalEvent<SignalButtonComponent, InteractHandEvent>(OnInteractHand);
         }
 
+        private void OnInit(EntityUid uid, SignalButtonComponent component, ComponentInit args)
+        {
+            var transmitter = EnsureComp<SignalTransmitterComponent>(uid);
+            if (!transmitter.Outputs.ContainsKey("Pressed"))
+                transmitter.AddPort("Pressed");
+        }
+
         private void OnInteractHand(EntityUid uid, SignalButtonComponent component, InteractHandEvent args)
         {
-            RaiseLocalEvent(uid, new InvokePortEvent("pressed"), false);
+            RaiseLocalEvent(uid, new InvokePortEvent("Pressed"), false);
```

Comment:
All the instances of "Pressed" and the like should probably be const strings. Would also help prevent bugs due to typos (e.g., prevent accidental lowercase "p"). 

Those consts should probably also be referenced in the default mapping list-tuple thing.